tests/js/spec/components/modals/widgetViewerModal.spec.tsx

```diff
@@ -819,81 +814,12 @@ describe('Modals -> WidgetViewerModal', function () {
     it('renders table header and body', async function () {
       await renderModal({initialData, widget: mockWidget});
       expect(screen.getByText('sum(sentry.sessions.session)')).toBeInTheDocument();
-      expect(screen.getByText('51k')).toBeInTheDocument();
-    });
-
-    it('renders pagination buttons', async function () {
-      await renderModal({initialData, widget: mockWidget});
-      expect(screen.getByRole('button', {name: 'Previous'})).toBeInTheDocument();
-      expect(screen.getByRole('button', {name: 'Next'})).toBeInTheDocument();
+      expect(screen.getByText('51292.954')).toBeInTheDocument();
```

Comment:
arg, did we regress with numeric shorthands in RH data? can you make a ticket for this, i can look into it!

Reply:
Hey @shruthilayaj , I don't think this a regression on RH. I just updated RH widgets in the viewer to display the full expanded value. This matches behaviour with Discover big number widgets in the WV as well. The feedback for this is that users want to see the full value while viewing in the widget viewer.

Let me know if you think there might still be an issue here though! I can create the ticket if needed.

Reply:
ps, changed the `51292.954` value in the text to `492` since i switched over to the sessions mock result `492` is the total from that mock.